charts/dave/charts/backend/values.yaml

```
@@ -109,16 +109,24 @@ extraVolumes:
 initContainers:
-  []
-  # - name: my-init-container-name
-  #   image: some-image:latest
-  #   env:
-  #     - name: AWS_DEFAULT_REGION
-  #       value: "eu-central-1"
-  #   volumeMounts:
-  #     - name: help-zammad
-  #       mountPath: /opt/zammad
-
+#  []
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)